src/ameba/cli/cmd.cr

```diff
@@ -51,6 +51,7 @@ module Ameba::Cli
     property except : Array(String)?
     property location_to_explain : NamedTuple(file: String, line: Int32, column: Int32)?
     property fail_level : Severity?
+    property? skip_reading_config = false
```

Comment:
Perhaps sth shorter?

```suggestion
    property? ignore_config = false
```

Reply:
Consider skip reading config is more accurately for what it does?